### DISCUSSION TITLE
Replace bind but keep using previous forwarders?

### DISCUSSION THREAD

**Discussion by idc77** [DNSCrypt/dnscrypt-proxy#2084] [Q&A]
On a Hetzner server with /etc/resolv.conf being
```
nameserver 185.12.64.1
nameserver 185.12.64.2
nameserver 2a01:4ff:ff00::add:1
nameserver 2a01:4ff:ff00::add:2
```
I'd like to replace the current bind nameserver with dnscrypt-proxy.
I do also and *especially* want to do this to offer (Chrome and Firefox as well as mobile) DOH.
Essentially, since my ISP in Germany is acting as a censor, I'd like to use my server as a DOH server using Hetzner's nameservers.
I'm not interested in the list of public servers.
I'd like to only use Hetzner's DNS and disalbe the rest.

How can I do that?

**Comment by jedisct1** [comment]
In order for your server to act as a DoH server, you need to install a DoH server: https://github.com/DNSCrypt/doh-server
